### PR TITLE
Fix for PR #5716 Move Logout button to end, and added name to CREDITS.md

### LINKED ISSUE #5716: [Feature] Move logout button to the end of list
**Is your feature request related to a problem? Please describe.**
I’m always frustrated<sup>1</sup> when I click the gear icon and accidentally log out. I also accidentally clicked logout from the settings page.

<sup>[1]: I love how the issue template started my sentence </sup>

**Describe the solution you’d like**
Please move logout to the last item in settings.  It is common practice, bet its part of some UX consensus---for example, github:
image

### PR DESCRIPTION
**Pull request by MananDhiman**
Closes #5716

How to test the feature manually:
1. After login, open nav menu using gear icon in top right and check at different resolutions and devices

Pull request checklist:
- [x] code manually tested

Preview
image

### CODE REVIEW

File: CREDITS.md [hunk 1]
```diff
@@ -239,3 +239,4 @@ People are sorted by name so please keep this order.
 * [yzqzss|一座桥在水上](https://github.com/yzqzss): [contributions](https://github.com/FreshRSS/FreshRSS/pulls?q=is:pr+author:yzqzss), [Web](https://blog.othing.xyz/)
 * [Zhaofeng Li](https://github.com/zhaofengli): [contributions](https://github.com/FreshRSS/FreshRSS/pulls?q=is:pr+author:zhaofengli), [Web](https://zhaofeng.li/)
 * [Zhiyuan Zheng](https://github.com/zhzy0077): [contributions](https://github.com/FreshRSS/FreshRSS/pulls?q=is:pr+author:zhzy0077)
+* [Manan Dhiman](https://github.com/manandhiman): [contributions](https://github.com/FreshRSS/FreshRSS/pulls?q=is:pr+author:manandhiman), [Web](https://manandhiman.com)
```
Comment (Alkarex) [inline CREDITS.md:242]: Please keep alphabetic order ;-)

**Comment by math-GH** [comment]
I am against this change. Sorry for that.
Let me explain.
I made the PR #3880 2 years ago that moved the logout link from bottom to top. Please see the reason there.

IMHO it is not a good UX, when the logout button is not well placed. The logout is connected to the user account, because the current user will be logged out (semantic logic).
If the user wants to logout, have the mouse way in mind: Before the PR (logout button is on the top of dropdown list): User clikc on the cog button and moves down the mouse a bit to click the logout button (about 50px). This PR makes the way much longer: The mouse has to move from top of the window to the bottom of the list (about 600px). If the user use it one his/her mobile device he/she needs (maybe) to scroll the list to reach logout button.

To be honest: I do not understand the issue why the logout button is a problem on the top of list next to the user account section.

Suggestion: If you have troubles with the logout button on the top: An extension could easily improve it for you.

Strategic discussion: I would like to improve the menu in general. If there is an interest I could make a draft that visualize some ideas.

**Comment by Alkarex** [comment]
I personally do not mind one way or the other.
In any case, independently of the default, we could suggest a CustomJS rule (maybe doable with CustomCSS and an `order` rule) for people of have strong feelings one way or the other

**Comment by Alkarex** [comment]
> I would like to improve the menu in general. If there is an interest I could make a draft that visualize some ideas.

👍🏻

**Comment by Alkarex** [comment]
@MananDhiman Sorry for the outcome, but your PR was just fine, so now you know how to do it for another time :-)

Examples of issues, which might not be too hard to implement:
* https://github.com/FreshRSS/FreshRSS/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22

**Comment by MananDhiman** [comment]
Perfectly fine, I'm just here to learn and practice. Always a next time.

Do I now close the pull request or something?

**Comment by math-GH** [comment]
As suggested: Here is my draft that includes a better logout button position: https://github.com/FreshRSS/FreshRSS/discussions/5966

**Pull request closed without merging**